javaws-wrapper/src/main/java/com/teamscale/jacoco/javaws/WindowsInstallation.kt

```diff
@@ -225,9 +224,9 @@ constructor(workingDirectory: Path) {
         }
     }
 
-    private inner class BackupPaths(private val backupDirectory: Path) {
-        private val securityPolicy: Path
-        private val ftypeMapping: Path
+    private inner class BackupPaths(internal val backupDirectory: Path) {
```

Comment:
Because private fields of inner classes are accessible to the surrounding class in Java, but are hidden from it in Kotlin. So I needed to make them internal.